index.js

```diff
@@ -48,6 +82,12 @@ var cacheControlNoTransformRegExp = /(?:^|,)\s*?no-transform\s*?(?:,|$)/
 function compression (options) {
   var opts = options || {}
 
+  if (hasBrotliSupport && opts.params === undefined) {
+    opts = copyShallow(opts)
+    opts.params = {}
+    opts.params[zlib.constants.BROTLI_PARAM_QUALITY] = 4
```

Comment:
This needs to be configurable by the user.

Reply:
> This needs to be configurable by the user.

It is. It's just a default (hinted by node.js docs but not set as the default. For some reason the highest level 11 is the default, which takes enormous amount of time with not a big size benefit.).
Level 4 is the sweet spot, kind of gzip's default just much better as it's brotli...

Reply:
Apologies! Please add to the documentation how the user can configure this, then.

Reply:
:-) Done!

Reply:
This is not resolved still. It seems like it's not possible to change the level still. If you disagree, please post an example program here that demonstrates this module with the level set to 1.

Reply:
You just pass them as `compression({ params: { [zlib.constants.BROTLI_PARAM_QUALITY]: 8 } })`.

Reply:
This piece of code just adds a reasonable default. That's it. It only does so if the `params` option is `undefined`.

Reply:
Wouldn't the user expect passing in `compression({ params: { foo: bar } })` still have a quality of 4 set? This does not seem to do that, and the documentation doesn't indicate that if you're passing any other params that the default quality is no longer 4. That is either a miss on the documentation or a miss on the code for that.

Reply:
It's a miss in the code after refactoring out the original shallow copy code. Adding a fix.